tests/test_actions.py

```diff
@@ -1255,3 +1255,53 @@ def test_diff__returns_result_of_injected_differs_diff_method(self):
         differ = Mock()
         result = self.actions.diff(differ)
         assert result == differ.diff.return_value
+
+    @patch("sceptre.plan.actions.StackActions._describe_stack_resource_drifts")
+    @patch("sceptre.plan.actions.StackActions._describe_stack_drift_detection_status")
+    @patch("sceptre.plan.actions.StackActions._detect_stack_drift")
+    def test_detect_stack_drift(
```

Comment:
I think you need tests for when drift detection times out and when it fails.

Reply:
Ok done.